src/components/Question/Answer.jsx

```diff
@@ -0,0 +1,9 @@
+import Button from '../Button/Button'
+
+const Answer = ({ answer, handleAnswerClick }) => (
```

Comment:
This component seems unnecessary to me, as it essentially calls the "Button" component.